lib/Controller/RoomController.php

```diff
@@ -2634,6 +2634,7 @@ public function scheduleMeeting(string $calendarUri, int $start, ?array $attende
 		$eventBuilder->setOrganizer($user->getEMailAddress(), $user->getDisplayName() ?: $this->userId);
 		$eventBuilder->setStartDate($startDate);
 		$eventBuilder->setEndDate($endDate);
+		$eventBuilder->setStatus('CONFIRMED');
```

Comment:
```suggestion
		if (method_exists($eventBuilder, 'setStatus')) {
			$eventBuilder->setStatus('CONFIRMED');
		}
```
if we want to backport